packages/radix-vue/src/Editable/EditableRoot.vue

```diff
@@ -207,10 +219,13 @@ provideEditableRootContext({
     @dismiss="handleDismiss"
   >
     <Primitive
+      v-bind="$attrs"
+      ref="primitiveElement"
       :as="as"
       :as-child="asChild"
       :dir="dir"
     >
+      {{ modelValue }}
```

Comment:
why we have additional `modelValue` here ya?

Reply:
My bad, I forgot to remove it. I used it for logging the value.